x-pack/plugins/session_view/public/components/process_tree_node/styles.ts

```diff
@@ -73,6 +73,8 @@ export const useStyles = ({ depth, hasAlerts, hasInvestigatedAlert, isSelected }
       '&:hover:before': {
         backgroundColor: hoverColor,
         transform: `translateY(-${size.xs})`,
+        marginLeft: '-50%',
```

Comment:
should this fix not go in the &:before bit below? I assume the other non hovered background treaments (investigated alert / selected process) will have the same issue? (top level seems fine. Seems it's the nested children that have the gap)

Can the below marginLeft and width calculation be fixed instead and account for the padding that is causing this? e.g  `calc(100% + <Container padding> + ${depth} * ${TREE_INDENT})

Reply:
<img width="1624" alt="Screen Shot 2022-04-06 at 11 02 12 PM" src="https://user-images.githubusercontent.com/8703149/162131105-c30a5e73-78f3-489b-869e-12f70d58e6bf.png">

i thought the issue only occur on hovered process ? no gap on selected process

Reply:
ya that one would be fine, it's a different codepath. The investigated alert highlight, and the alert red left border suffers from the same issue. 
![image](https://user-images.githubusercontent.com/16198204/162149262-171cce44-820d-4fb2-891f-78d89eb833cb.png)